### DISCUSSION TITLE
Does anyone have a fresh plugin for twind@1.0.0-next?

### DISCUSSION THREAD

**Discussion by tzakharko** [freshframework/fresh#665] [General]
I am currently playing around with the unreleased version of fresh, and the new plugin system looks very nice. I was wondering if anyone tried to patch the twind plugin for the yet unreleased twind@next? 

Thanks!

**Comment by sebilasse** [comment]
Please see https://github.com/denoland/fresh/discussions/317#discussioncomment-3488021
